yote.js

```diff
@@ -79,12 +79,12 @@ app.use(multipart({}));
 //only enable for development env - npm module can be buggy
 if (app.get('env') == 'development') {
   app.use(sass({
-    src: __dirname + '/public'
-    , dest: __dirname + '/public/css'
+    src: __dirname + '/client/public'
+    , dest: __dirname + '/client/public/css'
     , prefix: '/css'
     , debug: true
     , outputStyle: 'compressed'
-    , includePaths: ['public/app/', 'public/sass/globals/', 'globals/sass/includes/']
+    , includePaths: ['/client/public/app/', '/client/public/sass/globals/', 'globals/sass/includes/']
```

Comment:
I don't know the right way to do this. probably could put the SASS in the /client directory, but would need to output it to public, right?